### PR TITLE
Richard - Katlyn

### PR DESCRIPTION
**Pull request by bandizzzle**
Single-line Summary

Implemented functional programming, created a template for admin.html, and turned article.js and articleView.js into IIFEs.

Additional Info

First we started in articleView.js, and wrapped the entire page in IIFEs, and implemented functional programming.
Second, we created our template for admin.html.
Third, we moved onto the article.js page, and wrapped the entire page in IIFEs, and implemented functional programming.

[X] Our JavaScript passes our eslint.
[X] There is no extraneous, unrelated code included in this PR.
[X] We have titled the Pull Request with our names (ex: 'brian-rick').
[X] This PR includes commits from both myself and my partner; e.g. We followed good pair programming practices by switching driver/navigator roles.

### CODE REVIEW

File: starter-code/.eslintrc.json [hunk 1]
```diff
@@ -36,8 +36,8 @@
     "no-var": "warn",
     "prefer-const": "warn",
 
-    "eqeqeq": ["error", "always"],
-    "no-template-curly-in-string": "error",
+    // "eqeqeq": ["error", "always"],
```
Comment (Kdasay16) [inline starter-code/.eslintrc.json]: Why are you commenting these out?

File: starter-code/public/scripts/article.js [hunk 2]
```diff
@@ -1,112 +1,144 @@
...
 
-Article.fetchAll = callback => {
-    $.get('/articles')
-        .then(results => {
-            Article.loadAll(results);
-            callback();
+    // [X]TODO: Chain together a .map() and a .reduce() call to get a rough count of all words in all articles. Yes, you have to do it this way.
+    Article.numWordsAll = function () {
+        return Article.all.map(function (article) {
+            return article.body.split(' ').length;
```
Comment (Kdasay16) [inline starter-code/public/scripts/article.js]: Should not have a semicolon here

**Comment by Kdasay16** [comment]
There are a lot of extra semicolons throughout your code. Come see me if you would like to go through with me and delete them

File: starter-code/.eslintrc.json (hunk 1 above)
Comment (bandizzzle) [reply starter-code/.eslintrc.json]: It breaks my linter. My linter doesn't work at all and continues to give me an error message that says "EQEQEQ value must be not null". As soon as I comment it out and save it my linter works correctly. I asked SJ if that will effect anything and she said the EQEQEQ rule isn't important so commenting it out is ok.

**Pull request closed without merging**